robocop/checkers/comments.py

```diff
@@ -7,21 +7,31 @@
 from robot.utils import FileReader
 
 from robocop.checkers import RawFileChecker, VisitorChecker
-from robocop.rules import Rule, RuleSeverity
+from robocop.rules import Rule, RuleSeverity, RuleParam
 from robocop.utils import ROBOT_VERSION
 
 rules = {
     "0701": Rule(
+        RuleParam(
+            name="todos",
+            default="todo,fixme",
+            converter=str,
+            desc="List of case insensitive terms or phrases that violate the rule in comments.",
+        ),
         rule_id="0701",
         name="todo-in-comment",
-        msg="Found {{ todo_or_fixme }} in comment",
+        msg="Found a term or phrase {{ marker }} in the comments",
```

Comment:
Why not to use just:
```suggestion
        msg="Found '{{ marker }}' in the comments",
```
?
Also, it's better to add quotes here, not where the rule violation is reported.

Reply:
Good point. Makes code more clearer.